mgr/src/server/plugins/volume_expand.cr

```diff
@@ -211,15 +211,19 @@ put "/api/v1/pools/:pool_name/volumes" do |env|
 
   set_volume_metrics(req)
 
+  existing_nodes = participating_nodes(pool_name, volume)
+
+  # Remove duplicated node objects to avoid multiple node_actions to same node.
+  all_unique_nodes = unique_nodes(existing_nodes, nodes)
```

Comment:
```
all_unique_nodes = (existing_nodes + nodes).uniq { |item| item.name }
```

Reply:
Had seen this in doc, but had some error. So had implemented it manually. 
Will update.